source/info/bockowsk/Ebook.java

```diff
@@ -0,0 +1,54 @@
+package info.bockowsk;
+
+import java.io.*;
+
+class Ebook extends Book implements Comparable<Ebook> {
+
+    Ebook(String a, String t, String w) {
+        super(a,t,w);
+        String nazwa=a+"-"+t+".txt";
+        File plik1=new File(nazwa);
+        String linia;
+        try {
+            FileReader fileReader1=new FileReader(plik1);
+            BufferedReader reader1=new BufferedReader(fileReader1);
+            while ((linia=reader1.readLine())  != null ) {
+                super.tresc+=linia;
+            }
+        }
+        catch (FileNotFoundException ex1) {
+            ex1.printStackTrace();
+        }
+        catch (IOException ex2) {
+            ex2.printStackTrace();
+        }
+    }
+    public int compareTo(Ebook b) {
+        int b1=tresc.length();
+        int b2=b.tresc.length();
+        if (b1>b2) {
+            return -1;
+        }
+        else if (b1<b2) {
+            return 1;
+        }
+        else {
+            return 0;
+        }
+    }
+    public String toString() {
+        return autor+" : "+tytul;
+    }
+    public int hashCode() {
+        return tresc.hashCode();
+    }
+    public boolean equals(Object ebook) {
```

Comment:
Maintain consistent code format (i.e. there should be one blank line between methods).

Eclipse can help you and format the code for yourself. Try <kbd>Ctrl+Shift+F</kbd>. 
In IntelliJ it's <kbd>Ctrl+Alt+L</kbd>.

Reply:
Alright, I've just started using IDE (e.g. Eclipse), before I was using vim without any plugins. Thank you for a tip, I will use it every time before saving source code files.